tests/integration/instance-initializers/axe-component-test.js

```diff
@@ -122,7 +122,14 @@ test('audit should log any violations found', function(assert) {
   stubA11yCheck(sandbox, {
     violations: [{
       name: 'test',
-      nodes: []
+      nodes: [
```

Comment:
Can we add a test where there are no `nodes`? That will account for the `if (!nodes) {` branch added to the code above.